portfolio/src/main/webapp/style.css

```diff
@@ -90,6 +90,24 @@ body {
   width: 650px;
 }
 
+.main {
+  margin: auto auto;
+  width: 650px;
+  top: 120px;
+  position: relative;
+}
+
+.main.container {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+#hero {
+  padding-top: 6rem;
+  padding-bottom: 6rem;
+  animation: 1.4s down-enter cubic-bezier(.25,1,.5,1) backwards;
```

Comment:
This seems to be missing at least the animation-name property. Using the short-hand version for animation will need to follow a certain order. Alternatively you can specify each attribute separately, e.g. `animation-duration: 1.4s`

https://www.w3schools.com/cssref/css3_pr_animation.asp